### ISSUE TITLE
CI throws warnings to upgrade deprecated commands

### ISSUE THREAD

**Issue by Shinigami92** [faker-js/faker#1436]
Example: https://github.com/faker-js/faker/actions/runs/3234394475/jobs/5298373214#step:3:7

<details>
<summary>Details</summary>

```bash
Run pnpm/action-setup@v2.2.3
Warning: The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Running self-installer...
Installation Completed!

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
3s
Run actions/setup-node@v3
Found in cache @ /opt/hostedtoolcache/node/14.20.1/x[6](https://github.com/faker-js/faker/actions/runs/3234394475/jobs/5298373214#step:3:7)4

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
/home/runner/setup-pnpm/node_modules/.bin/pnpm store path --silent
/home/runner/.local/share/pnpm/store/v3
Warning: The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Received 38934331 of 5151[7](https://github.com/faker-js/faker/actions/runs/3234394475/jobs/5298373214#step:3:8)243 (75.6%), 37.1 MBs/sec
Received 51517243 of 51517243 (100.0%), 44.4 MBs/sec
Cache Size: ~49 MB (51517243 B)
/usr/bin/tar --use-compress-program unzstd -xf /home/runner/work/_temp/3dc62f5b-ac99-4291-b0d3-2b4d1507ac61/cache.tzst -P -C /home/runner/work/faker/faker
Cache restored successfully

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/[20](https://github.com/faker-js/faker/actions/runs/3234394475/jobs/5298373214#step:3:22)22-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Cache restored from key: node-cache-Linux-pnpm-269011b48e806205b548d1039aff355589fba7ba65975a15a8cdf6b3cc[21](https://github.com/faker-js/faker/actions/runs/3234394475/jobs/5298373214#step:3:23)0496
```

</details>

<img src="https://user-images.githubusercontent.com/7195563/195429419-b5c22596-9ef5-4dae-a081-bdc1f9ada3b5.png" width="400px"/>

**Comment by ST-DDT** [comment]
Will this be fixed by https://github.com/faker-js/faker/pull/1433 ?

**Comment by Shinigami92** [comment]
> Will this be fixed by #1433 ?

Having a look into their changes: https://github.com/pnpm/action-setup/compare/v2.2.2...v2.2.3
I highly assume: NO, it will not fix it at all

**Comment by xDivisionByZerox** [comment]
Looking at [current CI runs](https://github.com/faker-js/faker/actions/runs/4786369700/jobs/8510234480#step:3:1) this seems to no longer be the problem. 
@Shinigami92 can you confirm?
